arex-storage-web-api/src/main/java/com/arextest/storage/serialization/ZstdJacksonSerializer.java

```diff
@@ -26,6 +28,8 @@ public final class ZstdJacksonSerializer {
     public static final byte[] EMPTY_INSTANCE = SerializationUtils.EMPTY_INSTANCE;
     private final SerializationProvider serializationProvider;
 
+    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
> private static final ObjectMapper MAPPER = new ObjectMapper();

There is  a  instance of `SerializationProvider` created by `ZstdJacksonSerializer(ObjectMapper objectMapper)`, please remove `MAPPER `.